### DISCUSSION TITLE
How to group words of individual line

### DISCUSSION THREAD

**Discussion by joel1895** [mindee/doctr#1262] [Q&A]
Hi, the output seems complex. How can I group words in a single line.

**Comment by felixT2K** [comment]
Hi @joel1895 👋🏼,

If you know the page and line you need you can simply iterate trough the lines after exporting the results to json
`json_output = result.export()`
(https://mindee.github.io/doctr/using_doctr/using_models.html#what-should-i-do-with-the-output)

**Reply by troogyt** [reply]
`json_output = result.export()` does NOT produce JSON - the documentation you linked to above is incorrect and needs fixing. What it produces is a Python `dict` object, which is categorically different from true JSON.

See [#1377](https://github.com/mindee/doctr/issues/1377)

**Reply by felixT2K** [reply]
See the description above: 
`You can also export them as a nested dict, more appropriate for JSON format:`
